### ISSUE TITLE
Incorrect language used for a chart element

### ISSUE THREAD

**Issue by PowerGamer1** [amcharts/amcharts3#61]
Example:
```HTML
<!DOCTYPE html>
<html>
	<head>
		<title></title>
		<script src="http://www.amcharts.com/lib/3/amcharts.js" type="text/javascript"></script>
		<script src="http://www.amcharts.com/lib/3/serial.js" type="text/javascript"></script>
		<script src="http://www.amcharts.com/lib/3/lang/ru.js" type="text/javascript"></script>
		<script src="http://www.amcharts.com/lib/3/lang/fr.js" type="text/javascript"></script>
		<script type="text/javascript">
		
AmCharts.makeChart('chartdiv',{"type":"serial","language":"ru","categoryField":"created_at","dataDateFormat":"YYYY-MM-DD HH:NN:SS",
"categoryAxis":{"parseDates":true,"dateFormats":[{"period":"DD","format":"YYYY-MM-DD"},{"period":"MM","format":"YYYY-MM"},{"period":"YYYY","format":"YYYY"}]},
"graphs":[{"valueField":"score","bullet":"round"}],
"chartScrollbar":{"enabled":true},
"dataProvider":[{"created_at":"2014-08-20 15:47:38","score":"15.45"},{"created_at":"2014-09-04 15:48:31","score":"13.65"}],
});

AmCharts.makeChart('chartdiv2',{"type":"serial","language":"fr","categoryField":"created_at","dataDateFormat":"YYYY-MM-DD HH:NN:SS",
"categoryAxis":{"parseDates":true,"dateFormats":[{"period":"DD","format":"YYYY-MM-DD"},{"period":"MM","format":"YYYY-MM"},{"period":"YYYY","format":"YYYY"}]},
"graphs":[{"valueField":"score","bullet":"round"}],
"chartScrollbar":{"enabled":true},
"dataProvider":[{"created_at":"2014-08-20 15:47:38","score":"15.45"},{"created_at":"2014-09-04 15:48:31","score":"13.65"}],
});
		
		</script>
	</head>
	<body>
		<div id="chartdiv" style="height: 300px"></div>
		<div id="chartdiv2" style="height: 300px"></div>
	</body>
</html>
```
If you zoom the charts (by dragging the right grip of chart scrollbar to the left) until the "Show All" button appears in the top right corner of the chart you will see that it is displayed in 'fr' language for both charts but the first chart was configured to use 'ru' language.

**Comment by martynasma** [comment]
Seems like a bug. Thanks for the heads up.

We'll let you know when the fix is available.

P.S. moving forward, please contact amCharts support by contact@amcharts.com. Issues on GitHub are not being monitored by support staff as closely as the email helpdesk.

**Comment by martynasma** [comment]
Fixed in 3.10.3
